api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -1390,12 +1393,16 @@ public void testRenameNotebookinWorkspace() throws Exception {
     String nb1 = "notebooks/nb1.ipynb";
     String newName = "nb2.ipynb";
     String newPath = "notebooks/nb2.ipynb";
+    String fullPath = "gs://workspace-bucket/" + newPath;
+    String origFullPath = "gs://workspace-bucket/" + nb1;
     NotebookRename rename = new NotebookRename();
     rename.setName("nb1.ipynb");
     rename.setNewName(newName);
     workspacesController.renameNotebook(workspace.getNamespace(), workspace.getId(), rename);
     verify(cloudStorageService).copyBlob(BlobId.of(BUCKET_NAME, nb1), BlobId.of(BUCKET_NAME, newPath));
     verify(cloudStorageService).deleteBlob(BlobId.of(BUCKET_NAME, nb1));
+    verify(userRecentResourceService).updateNotebookEntry(1, 1, fullPath, Timestamp.from(NOW));
+    verify(userRecentResourceService).deleteNotebookEntry(1, 1, origFullPath);
```

Comment:
I just made this same comment in one of Neha's PRs. Can we use variables instead of literals in the tests (here and below)? I think it's easier to understand the test with names. Looking at these methods quickly, I'll have to do some hunting to understand the `1` values here to see if the test makes sense.